src/tools/rbd_mirror/ImageReplayer.cc

```diff
@@ -225,6 +225,19 @@ class ImageReplayerAdminSocketHook : public AdminSocketHook {
   Commands commands;
 };
 
+int replay_delay(const utime_t &event_time, int mirroring_delay) {
+    if (mirroring_delay <= 0) {
+      return 0;
+    }
+
+    utime_t now = ceph_clock_now(g_ceph_context);
```

Comment:
As I see, ceph_clock_now() uses clock_gettime(), which gets time since the Epoch, so it should not depend on the user timezone?

Also, not sure about using g_ceph_context here. In journal/Types.h I used nullptr, just not to add an additional header file from common. I think this should be consistent, i.e. nullptr here too. On the other hand, being able to shift time via clock_offset config option could be useful for testing, so may be use g_ceph_context here and when encoding?